src/recipes.jl

```diff
@@ -56,6 +56,16 @@ data space.
 - `nlabels_fontsize=labels_theme.fontsize`
 - `nlabels_attr=(;)`: List of kw arguments which gets passed to the `text` command
 
+### Inner node labels
+Put labels inside the marker. If labels are provided, change default attributes to
+`node_marker=Circle`, `node_strokewidth=1` and `node_color=:gray80`.
+The `node_size` will match size of the `ilabels`.
+
+- `ilabels=nothing`: `Vector` with label for each node
+- `ilabels_color=labels_theme.color`
+- `ilabels_fontsize=labels_theme.fontsize`
+- `ilabels_attr=(;)`: List of kw arguments which gets passed to the `text` command
```

Comment:
Perhaps we should note that `ilabels_attr` cannot be set interactively, and users should manipulate the underlying text plot directly!

Reply:
```
g = complete_graph(3)
visible = Observable(false)
graphplot(g; node_attr=(;visible))
visible[] = true #interactive update
```

Reply:
That would be the same as `nlabels_attr`, `elabels_attr`, `node_attr`, `edge_attr`, `arrow_attr`, right?

Probably this issue should be discussed separately?

Reply:
@asinghvi17 do you mean that `p = graphplot(...)`, `p[:ilabel_attr] = ...` doesn't work?

Reply:
> `_, _, p = graphplot(...), p[:ilabel_attr] = ...` doesn't work

I did mean that - the usage which @hexaeder suggested should work fine.  This wouldn't work because a reference to the attributes at the time of plot creation is splatted, and changing the attributes afterwards wouldn't have an effect.

I guess we should probably clarify that somewhere?   It's not a blocker for the PR by any means, just something I noticed.